src/main/java/com/example/Storehouse/responseBuilder/multiproduct/MultiProductResponseBuilder.java

```diff
@@ -0,0 +1,104 @@
+package com.example.Storehouse.responseBuilder.multiproduct;
+
+import com.example.Storehouse.entity.Products;
+import com.example.Storehouse.exception.exceptions.WrongQuantityException;
+import com.example.Storehouse.logger.Print;
+import com.example.Storehouse.models.product.Product;
+import com.example.Storehouse.models.product.ProductRequest;
+import com.example.Storehouse.models.request.MultiProductRequest;
+import com.example.Storehouse.models.response.multiProductsResponse.MultiProductsResponse;
+import com.example.Storehouse.service.StorehouseService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Component
+public class MultiProductResponseBuilder implements iMultiProductResponseBuilder {
```

Comment:
You can remove this class and migrate this to special Sevice class thst build all posible response classes

Reply:
done